SwiftRuntimeLibrary/SwiftMarshal/StructMarshal.cs

```diff
@@ -260,6 +260,19 @@ public IntPtr ProtocolWitnessof (Type t, Type withRespectTo = null)
 			return (IntPtr)pi.GetValue (null);
 		}
 
+		public IntPtr ProtocolConformanceof (Type interfaceType, Type withRespectTo)
```

Comment:
I realize it's not the first case but it still bug me to see a `of` suffix without capitalization